umodbus/functions.py

```diff
@@ -429,7 +430,7 @@ def float_to_bin(num: float) -> bin:
     :rtype:     bin
     """
     # no "zfill" available in MicroPython
-    # return bin(struct.unpack('!I', struct.pack('!f', num))[0])[2:].zfill(32)
+    # return bin(struct.struct.unpack('!I', struct.struct.pack('!f', num))[0])[2:].zfill(32)
```

Comment:
I assume there is one `struct` to much

Reply:
Thanks, I missed that part when I was adding the `struct` module back to the imported methoids `pack` and `unpack`.